pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.609.2</version><!-- which version of Jenkins is this plugin built against? -->
+        <version>2.6</version>
```

Comment:
Is this necessary?  This will require every user to be Jenkins 2.6 - not yet released.  And not everyone is upgrading to the very latest and greatest immediately.  Can a lower version be safely used?